src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -127,8 +127,9 @@ private ProcessPriorityClass PriorityClassCore
             {
                 EnsureState(State.HaveId);
 
-                int pri = Interop.libc.getpriority(Interop.libc.PriorityWhich.PRIO_PROCESS, _processId);
-                if (pri == -1 && Marshal.GetLastWin32Error() != 0)
+                int pri = 0;
+                int result = Interop.libc.getpriority(Interop.libc.PriorityWhich.PRIO_PROCESS, _processId, out pri);
+                if (result == -1)
```

Comment:
Shouldn't this be result != 0?  getpriority is returning Marshal.GetLastWin32Error(), so 0 would mean valid result and anything else would be an error, right?